src/picasso-components/custom-rect/index.js

```diff
@@ -0,0 +1,27 @@
+export default {
+  require: ['renderer'],
+  defaultSettings: {},
```

Comment:
Make sure to give all available settings a default value here. It will be easier if someone needs to make changes in here to see everything that can be configured from the outside.

Example:
```
defaultSettings: {
  settings: {
    type: 'max',
    extremum: () => undefined,
    separateStacking: false,
    orientation: 'horizontal',
    rtl: false,
  },
  style: {
    fill: '#999',
  },
},
```